velox/docs/functions/spark/string.rst

```diff
@@ -158,6 +158,11 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT replace('ABCabc', 'abc', ''); -- ABC
         SELECT replace('ABCabc', 'abc', 'DEF'); -- ABCDEF
 
+.. spark:function:: right(string, length) -> string
+
+    Returns the rightmost length characters from the ``string``.
+    If ``length`` is less or equal than 0 the result is an empty string.
```

Comment:
What is the behavior of length being larger that the size of string, could we clarify?

Reply:
@rui-mo Update the doc to indicate the original string will be returned.